### DISCUSSION TITLE
[BUG]- New-PnPBatch

### DISCUSSION THREAD

**Discussion by gregreda37** [pnp/powershell#3407] [General]
We are facing a New-PnPBatch issue when trying to initialize the variable, 

'<' is an invalid start of a value. Path $

Powershell version: 7.3.6

Has anybody faced this error before?

**Comment by westleyMS** [comment]
Check this issue, same error was caused by some cloud proxy-

[Get-PnPMicrosoft365GroupMember - EXCEPTION: '<' is an invalid start of a value. Path: $ | LineNumber: 0 | BytePositionInLine: 0](https://github.com/pnp/powershell/discussions/3012)
